apps/nextjs-website/react-components/components/Form/Form.tsx

```diff
@@ -36,9 +51,11 @@ const Form = (props: FormProps & { onSubmit: (data: FormData) => void }) => {
     organization: '',
   });
```

Comment:
```suggestion
    organization: '',
    newsletter: '',
  });
```
Why not include the value of the radio group in formData instead of having a whole separate state?